src/test/java/com/hillel/items_exchange/exceptionshandling/GlobalExceptionHandlerTest.java

```diff
@@ -5,13 +5,13 @@
 import com.hillel.items_exchange.controller.UserController;
 import com.hillel.items_exchange.dto.AdvertisementDto;
 import com.hillel.items_exchange.dto.CategoryDto;
+import com.hillel.items_exchange.dto.ChildDto;
 import com.hillel.items_exchange.dto.UserDto;
 import com.hillel.items_exchange.exception.DataConflictException;
+import com.hillel.items_exchange.exception.EntityAmountException;
 import com.hillel.items_exchange.exception.IllegalOperationException;
 import com.hillel.items_exchange.exception.handler.GlobalExceptionHandler;
-import com.hillel.items_exchange.util.AdvertisementDtoCreatingUtil;
-import com.hillel.items_exchange.util.CategoryTestUtil;
-import com.hillel.items_exchange.util.UserDtoCreatingUtil;
+import com.hillel.items_exchange.util.*;
```

Comment:
Why do you import all classes from `com.hillel.items_exchange.util`?

Reply:
I'm not on purpose =) I will fix this

Reply:
@lubchik-14 We need 5 out of 7 imports from this package, maybe should be better to leave it as it is?

Reply:
Maybe someone else can help us?